documentation/docs/install-pmm/install-pmm-server/deployment-options/docker/easy-install.md

```diff
@@ -1,8 +1,11 @@
-## Run Docker via the Easy-install script
-!!! caution alert alert-warning "Caution"
-    You can download and check the installation script before running it from our [Github](https://www.percona.com/get/pmm):
+# Run Docker via the Easy-install script
 
-## Linux or macOS
+## Security best practice
+You can download and check the installation script before running it from our [Github](https://www.percona.com/get/pmm).
```

Comment:
Considering the link isn't an actual direct GitHub link, do you think we should remove that bit?
```suggestion
You can download and check the [installation script](https://www.percona.com/get/pmm) before running it.
```